Vagrantfile

```diff
@@ -49,13 +49,17 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |box|
     box.memory = 1024
     box.cpus = 1
+    box.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
   end
 
+  config.vm.provision :shell, :inline => 'apt-get update'
```

Comment:
this shouldn't be done here, it belongs in the provisioning code for the project. fairly certain the startupgenome chef code you're working with does this first thing, but if it doesn't it should